rust-toolchain

```diff
@@ -1 +1 @@
-1.51.0
+1.62.1
```

Comment:
Is the upgrade strictly needed. I usually try to keep the version low and only bump if if needed. For Rust edition 2021, 1.56.1 should be enough.